### PR TITLE
Jasmine/add api

### PR DESCRIPTION
**Pull request by Yaqing-Peng**
added and implemented all api for menu, order, and user

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,6 +29,33 @@
...
         </dependency>
+
+        <!--mybatis 起步依赖-->
+        <dependency>
+            <groupId>org.mybatis.spring.boot</groupId>
+            <artifactId>mybatis-spring-boot-starter</artifactId>
+            <version>3.0.2</version>
+        </dependency>
+
+        <!-- mysql驱动包 , 刚刚发布的最新版本的驱动包-->
```
Comment (ShawFengZ) [inline pom.xml:40]: If you need comments, then comment with English

File: pom.xml [hunk 2]
```diff
@@ -27,6 +29,33 @@
...
+<!--        <dependency>-->
```
Comment (ShawFengZ) [inline pom.xml:41]: Remove unused code

File: src/main/resources/mybatis-generator-config.xml [hunk 3]
```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--mybatis的代码生成器相关配置-->
```
Comment (ShawFengZ) [inline src/main/resources/mybatis-generator-config.xml:2]: English or remove

File: pom.xml (hunk 1 above)
Comment (Yaqing-Peng) [reply pom.xml:40]: ok

File: pom.xml (hunk 2 above)
Comment (Yaqing-Peng) [reply pom.xml:41]: ok

File: src/main/resources/mybatis-generator-config.xml (hunk 3 above)
Comment (Yaqing-Peng) [reply src/main/resources/mybatis-generator-config.xml:2]: ok

**Review by ShawFengZ — APPROVED** [review]

**Review by ShawFengZ — APPROVED** [review]

**Pull request merged**